src/processor/operator/persistent/reader/csv/driver.cpp

```diff
@@ -111,6 +111,56 @@ common::DataChunk& getDummyDataChunk() {
     return dummyChunk;
 }
 
+SniffCSVDialectDriver::SniffCSVDialectDriver(SerialCSVReader* reader,
+    const function::ScanTableFuncBindInput* bindInput)
+    : SerialParsingDriver(getDummyDataChunk(), reader) {
+    auto& csvOption = reader->getCSVOption();
+    (void)bindInput;
+    column_counts = std::vector<idx_t>(csvOption.sampleSize, 0);
+}
+
+bool SniffCSVDialectDriver::addValue(uint64_t rowNum, common::column_id_t columnIdx,
+    std::string_view value) {
+    (void)rowNum;
+    (void)columnIdx;
+    (void)value;
```

Comment:
Why do we have to do this?
If a parameter is unused, you can do (for example: rowNum is unused):
```
SniffCSVDialectDriver::addValue(uint64_t /*rowNum*/, common::column_id_t columnIdx,
    std::string_view value)
```